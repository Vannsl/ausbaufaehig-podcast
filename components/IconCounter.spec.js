import { shallowMount } from "@vue/test-utils";
import IconCounter from "./IconCounter";

describe("IconCounter", () => {
  const wrapper = shallowMount(IconCounter);

  it("renders the correct markup", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
