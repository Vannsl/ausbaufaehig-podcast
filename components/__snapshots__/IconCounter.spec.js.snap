// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconCounter renders the correct markup 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
